src/datepicker/datepicker-navigation.ts

```diff
@@ -38,6 +38,15 @@ import {NgbDatepickerI18n} from './datepicker-i18n';
         <span class="ngb-dp-navigation-chevron"></span>
       </button>
     </div>
+    <span class="sr-only" [attr.aria-live]="'polite'" [attr.aria-atomic]="true">
+      <span *ngIf="months.length > 1">
+        <span i18n="@@ngb.datepicker.screenreader.from">from</span>
+        {{ this.getScreenReaderFrom() }}
+        <span i18n="@@ngb.datepicker.screenreader.to">to</span>
+        {{ this.getScreenReaderTo() }}
```

Comment:
I think I would name the `i18n` keys with `screenreader` inside. Just call it the way it should, then if we use it in SR context, it seems to be detail implementation.

Reply:
Removed "screenreader" from i18n tag names.